src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,117 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(nullable = true)
+    private String genre;
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false)
+    private String image;
+    @Column(nullable = false)
+    private String title;
+    @Column(nullable = false)
+    private String subtitle;
+    @Column(nullable = false)
+    private String publisher;
+    @Column(nullable = false)
+    private String year;
+    @Column(nullable = false)
+    private Integer pages;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+
+    public Book() {
+
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```

Comment:
The id setter shouldn't be defined unless exctrictly necessary and justified, given these ids will be sorted out by JPA.

Reply:
Perfect, I start with the changes :+1: